src/client/Microsoft.Identity.Client/Platforms/iOS/Broker/iOSBroker.cs

```diff
@@ -47,7 +47,7 @@ public iOSBroker(ICoreLogger logger, ICryptographyManager cryptoManager, CoreUIP
             _uIParent = uIParent;
         }
 
-        public bool IsBrokerInstalledAndInvokable()
+        public bool IsBrokerInstalledAndInvokable(AuthorityType authorityType = AuthorityType.Aad)
```

Comment:
Discussed offline - would be clearer if this param didn't have a default value and so callers will pass the actual type. Android and iOS brokers would just ignore the type in their implementation of `IsBrokerInstalledAndInvokable` (i.e. only the signature changes, but the logic stays the same), to minimize the changes .

Reply:
Ok, made changed